apps/api/e2e/utils/test-helpers.ts

```diff
@@ -254,34 +238,6 @@ export async function startExistingTestCompetition(
   return result as StartCompetitionResponse;
 }
 
-/**
- * Ensure the database is initialized before using it
- */
-async function ensureDatabaseInitialized(): Promise<void> {
-  if (!isDatabaseInitialized) {
-    console.log("Initializing database for tests...");
-    await dbManager.initialize();
-    isDatabaseInitialized = true;
-  }
-}
-
-/**
- * Clean up database state for a given test case
- * This can be used in beforeEach to ensure a clean state
- * Now delegated to the DbManager for consistency
- */
-export async function cleanupTestState(): Promise<void> {
-  await ensureDatabaseInitialized();
-
-  // Also reset rate limiters to ensure clean state between tests
-  resetRateLimiters();
```

Comment:
Here's another spot were the sub-process rate limiters are the only thing we can about, but since this is interacting with a Service instance that is in the test runner this call to `resetRateLimiters` doesn't do anything.